### PR TITLE
Document APNS platform value

### PR DESCRIPTION
**Pull request by benbz**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,6 +54,10 @@ apns
     * the 'team_id' parameter
     * the 'topic' parameter
 
+  For either type:
+    It can accept:
+    
+    * the 'platform' parameter which determines whether the production or sandbox APNS environment is used. Valid values are 'production' or 'sandbox'. If not provided, 'production' is used
```
Comment (babolivier) [inline README.rst:60]:
```suggestion
    * the 'platform' parameter which determines whether the production or sandbox APNS environment is used. Valid values are 'production' or 'sandbox'. If not provided, 'production' is used.

```

**Comment by benbz** [comment]
Thanks! Who merges this - me now, you/other at some point in time?

**Comment by babolivier** [comment]
Usually it's the PR's author, but I'm happy to merge if you prefer that I do it (if you do it, please use "Squash and merge" rather than the normal merge though).

**Comment by benbz** [comment]
I'm unsure if that's done the right thing. I selected squash & merge but then it failed initially. I'm unclear whether the try again button has done it correctly

**Pull request merged**